README.md

```diff
@@ -1,2 +1,58 @@
 # koa-npm-resolution
-Koa middleware that transforms NPM package specifiers to relative paths
+
+This is a middleware for Koa servers that resolves NPM package specifiers in ES modules to relative paths for use on the web.  It is currently only intended for use as middleware in development servers to fascilitate build-free testing/iteration.
```

Comment:
personal pedantic nit: I say "standard JS modules" because we don't say "ES Classes" and "ES const" etc

Reply:
Updated!